src/store/storage-controller.ts

```diff
@@ -78,6 +80,9 @@ export class StorageController {
     }
 
     await messageStore.put(tenant, message, indexes);
+
+    const messageCid = await Message.getCid(message);
+    await eventLog.append(tenant, messageCid.toString());
```

Comment:
Nit: `messageCid` is already a string.
```suggestion
    await eventLog.append(tenant, messageCid);
```